src/collection.js

```diff
@@ -242,6 +242,15 @@ export default class Collection {
   }
 
   /**
+   * Closes any current connection to the database for this collection.
+   *
+   * @return {Promise}
+   */
+  close() {
+    return this.db.close();
+  }
+
+  /**
```

Comment:
Is it really necessary for now? I can't see much issue with implicit opening, you already get a rejection from the adapter when trying to execute a query with no opened db…